### PR TITLE
Adds space funeral mobs

### PR DESCRIPTION
**Pull request by Blute1**
:cl: Spacedong
add: Added space funeral mobs. Currently admin-spawn and goldcore spawn only.
/:cl:

### CODE REVIEW

**Comment by PopNotes** [comment]
i'm shitting

**Comment by Blute1** [comment]
ur a real G if u remember these mobs ayy

File: hippiestation.dme [hunk 1]
```diff
@@ -735,10 +735,10 @@
...
 #include "code\game\objects\effects\decals\misc.dm"
 #include "code\game\objects\effects\decals\remains.dm"
-#include "code\game\objects\effects\decals\Cleanable\aliens.dm"
-#include "code\game\objects\effects\decals\Cleanable\humans.dm"
-#include "code\game\objects\effects\decals\Cleanable\misc.dm"
-#include "code\game\objects\effects\decals\Cleanable\robots.dm"
+#include "code\game\objects\effects\decals\cleanable\aliens.dm"
+#include "code\game\objects\effects\decals\cleanable\humans.dm"
+#include "code\game\objects\effects\decals\cleanable\misc.dm"
+#include "code\game\objects\effects\decals\cleanable\robots.dm"
```
Comment (KayeArray) [inline hippiestation.dme:748]: ???

**Comment by dreamcrab** [comment]
yes +55

**Comment by McBawbaggings** [comment]
yes
meatpizza too

**Comment by Dudeinyourbackyard** [comment]
fuck yea +1 when everything is good
bring back meat pizza too
one that doesn't crash the server

**Comment by Blute1** [comment]
ready to merge

**Comment by PopNotes** [comment]
build this

**Comment by banthebantz** [comment]
+dongcode

**Comment by tortellinitony** [comment]
they're also ugly as fuck

**Comment by Dudeinyourbackyard** [comment]
das how they sposed to be shitter

**Comment by Blute1** [comment]
ill fix this tmrw

**Comment by Blute1** [comment]
heres a full sprite map

**Comment by WallaceShark** [comment]
+1. This is sure to give the xenobiologist players a good shock!

**Comment by 2Dollarbill** [comment]
I remember these, they were old as hell but I'd gladly accept them again.

**Comment by Chnkr** [comment]
Your PR needs to be updated to reflect the current state of the master branch, please update your PR and either (re)open it, or ask a maintainer to (re)open it.

**Pull request closed without merging**